addons/product/models/product_product.py

```diff
@@ -242,6 +242,9 @@ def _check_barcode_uniqueness(self):
             self._check_duplicated_product_barcodes(barcodes_within_company, company_id)
             self._check_duplicated_packaging_barcodes(barcodes_within_company, company_id)
 
+    def _check_edi_move(self):
```

Comment:
As mentioned by NLE, I'd move this into `account`. Also I find the method name slightly confusing. Could you replace it with something like `_check_edi_line_tax_required` (also taken from NLE's comment).

Reply:
Done :+1: